docs/components/best-practices/development/understanding-transaction-handling-c7.md

```diff
@@ -92,27 +92,27 @@ Aside a general strategy to mark service tasks as being save points you will oft
 
 - _User tasks_ <img src="/img/bpmn-elements/task-user.svg" width="40"/>: This savepoint allows users to complete their tasks without waiting for expensive subsequent steps and without seeing an unexpected rollback of their user transaction to the waitstate before the user task. Sometimes, e.g. when validating user input by means of a subsequent step, you want exactly that: rolling back the user transaction to the user task waitstate. In that case you might want to introduce a savepoint right after the validation step.
```

Comment:
<img width="995" alt="image" src="https://github.com/user-attachments/assets/66f41b77-7709-42d8-b926-c88af55c71e9">

This one is also not visible on the page.

Reply:
added the missing ones.